### ISSUE TITLE
WIP: Tab order/behavior

### ISSUE THREAD

**Issue by je-boska** [substrate-developer-hub/substrate-docs#1192]
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Bug report for the website

**The tab order skips the following page elements:**

- Header nav search
- Color mode
- Header nav children with sub-menus (Opportunities, Resources, Connect)
- Banner close icon

**Following page elements are selected with tabbing, but have no focus styling:**
 - Header nav items (Technology, Developers, Vision, Ecosystem)
 - Sidebar nav open/close icon

**Other tab order issues:** 
 - Banner is selected first
 - When using sidebar nav, tabbing position is reset on click
 - Table of Contents nav is after article in tab order
 - Footer: Social media icons are before Privacy Policy and ToS in tab order

This ticket will be updated as more issues are identified.

### Steps to reproduce the problem

Toggle through the page elements using tab, shift+tab and enter.

**Comment by jonnysmillie** [comment]
I have begun to address this on a branch: https://github.com/substrate-developer-hub/substrate-docs/tree/issue/tab-order

This should fix

- Header nav items (Technology, Developers, Vision, Ecosystem)
- Header nav children with sub-menus (Opportunities, Resources, Connect)
- Banner close icon

---

In regard to these:

**Other tab order issues:**

- Banner is selected first 
- When using sidebar nav, tabbing position is reset on click
- Table of Contents nav is after article in tab order
- Footer: Social media icons are before Privacy Policy and ToS in tab order

As far as I know this is all expected behaviour so unless you can make a good case for any alternative I will ignore.

---

I will return to address:

- Header nav search
- Color mode
- Sidebar nav open/close icon
